x-pack/plugins/apm/public/application/csmApp.tsx

```diff
@@ -4,52 +4,51 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
```

Comment:
@shahzad31 not as many changes here as it may look; I'm just reordering imports with option-shift-o.